packages/kbn-securitysolution-grouping/src/hooks/use_get_group_selector.tsx

```diff
@@ -110,20 +112,25 @@ export const useGetGroupSelector = ({
 
   const onChange = useCallback(
     (groupSelection: string) => {
-      if (selectedGroups.find((selected) => selected === groupSelection)) {
-        const groups = selectedGroups.filter((selectedGroup) => selectedGroup !== groupSelection);
-        if (groups.length === 0) {
-          setSelectedGroups(['none']);
-        } else {
-          setSelectedGroups(groups);
+      // Simulate a toggle behavior when maxGroupingLevels is 1
+      if (maxGroupingLevels === 1) {
+        setSelectedGroups([groupSelection]);
```

Comment:
this looks to be the only untested condition here

Reply:
test added 🙌